distributed/client.py

```diff
@@ -1313,8 +1313,7 @@ async def _start(self, timeout=no_default, **kwargs):
         for topic, handler in Client._default_event_handlers.items():
             self.subscribe_topic(topic, handler)
 
-        for preload in self.preloads:
-            await preload.start()
+        await self.preloads.start()
```

Comment:
There's an argument to be made that `preload.start` should raise here. Thoughts?

Reply:
Agreed; I don't think it's healthy to semi-quietly go online nonetheless.